pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/CreateUserActivityUploadRoute.java

```diff
@@ -117,9 +117,10 @@ public Object handle(Request request, Response response, CreateUserActivityUploa
                     operatorUser.getId(),
                     instanceDto.getParticipantId(),
                     fileQuestionDef,
-                    String.format("%s/%s", studyGuid, instanceDto.getActivityCode()),
+                    String.format("%s/%s_%s_%s_%s_%d", studyGuid, instanceDto.getActivityCode(), userGuid,
```

Comment:
This one changes ddp-upload/STUDY_NAME/ACTIVITY_NAME/FILE_NAME to ddp-upload/STUDY_NAME/ACTIVITY_NAME_FILE_NAME

Reply:
What are your thoughts on moving this to a private `getBlobName()` method call? Using the String.format call here feels unwieldy and breaks context for me.